.circleci/config.yml

```diff
@@ -0,0 +1,84 @@
+# These environment variables must be set in CircleCI UI
+#
+# GCLOUD_SERVICE_KEY - key for pushing images to gcr
+# GOOGLE_PROJECT_ID - moz-fx-data-airflow-prod-88e0
+# GOOGLE_COMPUTE_ZONE - us-west1-b
+#
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:18.02.0-ce
+    working_directory: ~/mozilla/probe-scraper
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: docker build -t app:build .
+
+  test:
+    docker:
+      - image: python:3.7
+    working_directory: ~/mozilla/probe-scraper
+    steps:
+      - checkout
+      - run: pip install -r requirements.txt -r test_requirements.txt
```

Comment:
We definitely want these tests to run _inside_ the docker container. As such we'll need to add the `bin/entrypoint` file and run them using `docker run app:build test`.